src/Spout/Writer/Common/Entity/Cell.php

```diff
@@ -164,4 +202,18 @@ public function __toString()
     {
         return (string)$this->value;
     }
+
+    /**
+     * @param Style $style|null
+     * @return $this
+     */
+    public function applyStyle(Style $style = null)
```

Comment:
This should be moved to a CellManager: `CellManager::applyStyle(Cell $cell, Style $style = null)`
The idea is to keep entities logic-less, as much as possible. The logic should live in (as much as possible) stateless managers.